libnl-route-3.sym

```diff
@@ -1340,5 +1340,9 @@ global:
 	rtnl_link_bond_get_miimon;
 	rtnl_link_bond_get_min_links;
 	rtnl_link_bond_get_mode;
+	rtnl_link_bridge_get_nf_call_arptables;
+	rtnl_link_bridge_get_nf_call_ip6tables;
+	rtnl_link_bridge_get_nf_call_iptables;
+	rtnl_link_bridge_set_vlan_default_pvid;
```

Comment:
Optimally, this bugfix would be a separate commit. Could you be bothered to split it out? :)

Reply:
Sure